source/jormungandr/jormungandr/interfaces/v1/Uri.py

```diff
@@ -119,7 +121,7 @@ def get(self, region=None, lon=None, lat=None, uri=None, id=None):
         if args.get("headsign"):
             f = u"vehicle_journey.has_headsign({})".format(protect(args["headsign"]))
             if args.get("filter"):
-                args["filter"] += " and " + f
+                args["filter"] = '({}) and {}'.format(args["filter"], f)
```

Comment:
Minor: This "headsign" param should probably be grouped and handled in get_filters.